app/src/main/java/net/sourceforge/opencamera/ui/RecSyncView.java

```diff
@@ -0,0 +1,82 @@
+package net.sourceforge.opencamera.ui;
+
+import net.sourceforge.opencamera.MainActivity;
+import net.sourceforge.opencamera.MyDebug;
+import net.sourceforge.opencamera.R;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.Button;
+import android.widget.LinearLayout;
+
+import androidx.annotation.StringRes;
+
+import com.googleresearch.capturesync.SoftwareSyncController;
+
+/**
+ * This defines the UI for the "recSync" button, that provides quick access to a
+ * range of options.
+ * <p>
+ * Requires {@link SoftwareSyncController} to be initialized (i.e. RecSync to be started)
+ */
+public class RecSyncView extends LinearLayout {
+    private static final String TAG = "RecSyncView";
+
+    public static final float ALPHA_BUTTON_SELECTED = 1.0f;
+    public static final float ALPHA_BUTTON = 0.6f; // 0.4f tends to be hard to see in bright light
+
+    private final float scale = getResources().getDisplayMetrics().density;
+    private int total_width_dp;
+
+    private final Button button_sync_settings;
+    private final Button button_align_phases;
+
+    public RecSyncView(Context context) {
+        super(context);
+
+        if (MyDebug.LOG) {
+            Log.d(TAG, "new RecSyncView: " + this);
+        }
+
+        this.setOrientation(LinearLayout.VERTICAL);
+
+        final MainActivity main_activity = (MainActivity) this.getContext();
+
+        boolean small_screen = false;
+        total_width_dp = 280;
+        int max_width_dp = main_activity.getMainUI().getMaxHeightDp(false);
```

Comment:
What do youi want here? View has layout params and omMeasure method if you want to control its size by maximal width or smth like that

Reply:
This is the way widgets like this were implemented originally and we decided to do it the same way to avoid any possible problems

Reply:
oh, really strange, okey lets skip this